File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    answer = 0
    v = binomial.split()
    v0 = int(v[0])
    v2 = int(v[2])
    if v[1] == "+":
        answer = v0+v2
    elif v[1] == "-":
        answer = v0-v2
    else:
        answer = v0*v2
    return answer

File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.js
function solution(binomial) {
    let aopb = binomial.split(' ');
    const a = parseInt(aopb[0]);
    const b = parseInt(aopb[2]);
    const op = aopb[1];
    
    switch(op){
        case '+':
            return a+b;
            break;
        case '-':
            return a-b;
            break;
        case '*':
            return a*b;
            break;
        case '/':
            return parseInt(a/b);
            break;
        default:
            break;
    }
}